README.md

```diff
@@ -60,7 +60,3 @@ mailing list. You are very welcome to join.
 If you are new to contributing to open source, [this
 guide](https://opensource.guide/how-to-contribute/) helps explain why, what,
 and how to successfully get involved.
-
```

Comment:
This still needs to be reverted.

Reply:
@rkern I guess that portion was prior to the starting of this thread.